software/docker/engine-rpm-pub/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.7
+
+WORKDIR /usr/app
+
+COPY requirements.txt ./
+
+RUN pip install --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt
+
+COPY ./engine-rpm-pub.py .
+COPY ./raw-can.avsc .
+COPY ./start.sh .
+
+CMD ["./start.sh"]
```

Reference: comment on `start.sh`

```suggestion
CMD ["python engine-rpm-pub.py"]
```